common/app/model/dotcomrendering/pageElements/PageElement.scala

```diff
@@ -23,7 +25,8 @@ case class ImageSource(weighting: String, srcSet: Seq[SrcSet])
 case class AudioBlockElement(assets: Seq[AudioAsset]) extends PageElement
 case class GuVideoBlockElement(assets: Seq[VideoAsset], imageMedia: ImageMedia, data: Map[String, String], role: Role) extends PageElement
 case class VideoBlockElement(data: Map[String, String], role: Role) extends PageElement
-case class EmbedBlockElement(html: Option[String], safe: Option[Boolean], alt: Option[String], isMandatory: Option[Boolean]) extends PageElement
+case class EmbedBlockElement(html: String, safe: Option[Boolean], alt: Option[String], isMandatory: Boolean) extends PageElement
+case class SoundcloudBlockElement(html: String, track: Option[String], playlist: Option[String], isMandatory: Boolean) extends PageElement
```

Comment:
Maybe we want `id` and some kind of track/playlist flag instead https://github.com/guardian/dotcom-rendering/pull/317#discussion_r240304479